source/app/blueprints/manage/templates/modal_add_asset_type.html

```diff
@@ -10,24 +10,66 @@ <h4>Add Asset Type</h4>
             <div class="col-md-12 col-lg-12 col-sm-12">
                 {{ form.hidden_tag() }}
                 <div class="form-group">
-                    <label for="asset_type" class="mr-4">Type
+                    <label for="asset_type" class="mr-4">Assett Type
                     </label>
                     {{ form.asset_name(class='form-control',  autocomplete="off") }}
                 </div>
                 <div class="form-group mt-3">
-                    <label for="asset_description" class="placeholder">Type description</label>
+                    <label for="asset_description" class="placeholder">Asset Type description</label>
                     {{ form.asset_description(class='form-control',  autocomplete="off") }}
                 </div>
+                <div class="row ml-2">
+                {% if assettype.asset_icon_not_compromised %}
+                    <div class="form-group col-2">
+                        <label for="asset_icon_not_compromised" class="mr-4">Preview</label>
+                        <div id="asset_icon_not_compromised">
+                            <img style="widht:2em;height:2em" src="/static/assets/img/graph/{{ assettype.asset_icon_not_compromised }}">
```

Comment:
I am thinking about storing the icons in a persistent volumen in `/home/iris/asset_icons` and symlinking them into the `/static/assets/img/graph/` folder when writing them to disk. 
When the container is recreated, I will need to run a quick bash command to re-symlink all the old asset icons.
@whikernel  do you think that could work or is there a issue with that?

Reply:
That should probably do it. Pathlib with `symlink_to` could be used to stay in full python. This can be added to the post_init. 
For the docker volume, could you please use something generic like `- server_data:/home/iris/server_data`, and then creates a subfolder like `custom_assets` ? An upcoming change will also requires a persistent volume and we're creating too many. We'll migrate `iris-downloads` and `user_templates` at some points too.

Reply:
Thanks for the suggestions, I will get that sorted.

Reply:
Fixed to dynamic paths, added generic `server_data` volume with `custom_assets` subfolder.